### PR TITLE
Enforce expectation names uniqueness in CheckResultOutline test

### PR DESCRIPTION
**Pull request by nelsonkopliku**
# Description

This should remove flakiness for the specific test changed.

### CODE REVIEW

**Review by EMaksy — APPROVED** [review]
Yeah it's a lot more consistent this way :D
LGTM :D

**Review by dottorblaster — APPROVED** [review]
LGTM but since those are strings, could we just use uuids?

File: assets/js/components/ExecutionResults/CheckResultOutline.test.jsx [hunk 1]
```diff
@@ -43,11 +43,11 @@ describe('CheckResultOutline Component', () => {
     const checkID = faker.datatype.uuid();
     const clusterName = faker.lorem.word();
 
-    const expectationName1 = faker.company.name();
-    const expectationName2 = faker.color.human();
-    const expectationName3 = faker.color.human();
-    const expectSameExpectationName1 = faker.lorem.word();
-    const expectSameExpectationName2 = faker.lorem.word();
+    const expectationName1 = faker.datatype.uuid();
```
Comment (nelsonkopliku) [inline assets/js/components/ExecutionResults/CheckResultOutline.test.jsx:46]: @dottorblaster I applied the suggested change about uuids.
 
I am afraid, though, that the _should expectation names be  uuids?_ doubt might rise, which is not the case. 

As long as it clear that we just want strings that are unique in a restricted context and not necessarily uuids, we should be good.

Comment (dottorblaster) [reply assets/js/components/ExecutionResults/CheckResultOutline.test.jsx:46]: Yeah I agree, for me it's clear, I don't know if you want to leave a comment or something :sweat_smile:

**Pull request merged**